compiler/testData/codegen/bytecodeText/lineNumbers/ifConsts.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND_FIR: JVM_IR
```

Comment:
this shifts the line numbers, so either the line needs to be at the bottom or the expected results should be updated

Reply:
Fixed, thank you.

Reply:
BTW I unmuted 15 FIR bytecode text tests in process (they were muted directly because of this line shift).

Reply:
Oh sorry about that! I should've known better. 🙁  Thanks @pyos for the callout and @mglukhikh for the quick fix!